sklearn/decomposition/pca.py

```diff
@@ -112,7 +112,8 @@ class PCA(_BasePCA):
 
     It uses the LAPACK implementation of the full SVD or a randomized truncated
     SVD by the method of Halko et al. 2009, depending on the shape of the input
-    data and the number of components to extract.
+    data and the number of components to extract. The input data is centered
```

Comment:
Maybe move that to the first paragraph above? Otherwise seems good.

Reply:
Done